dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -321,6 +321,30 @@ public Tool updateContainer(@ApiParam(hidden = true) @Auth User user,
 
     }
 
+    @PUT
+    @Timed
+    @UnitOfWork
+    @Path("/{toolId}/defaultVersion")
+    @ApiOperation(value = "Update the default version of the given tool.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Tool.class)
+    public Tool updateDefaultVersion(@ApiParam(hidden = true) @Auth User user,
+                                @ApiParam(value = "Tool to modify.", required = true) @PathParam("toolId") Long toolId,
+                                @ApiParam(value = "Tag name to set as default.", required = true) String tag) {
+        Tool t = toolDAO.findById(toolId);
```

Comment:
Ironically, I just did something similar in #1574 
 (DRY)
Stuff this copy-pasted code in EntryVersionHelper.java for now